Makefile

```diff
@@ -55,3 +55,9 @@ Leaflet.markercluster:
 
 Leaflet.awesome-markers:
 	$(GIT_SUBTREE) --prefix=lib/Leaflet.awesome-markers https://github.com/lvoogdt/Leaflet.awesome-markers.git 2.0/develop
+
+docs:
```

Comment:
We don't really use this anymore.  Check out `routes/dev-console.php`.  The plan is to start merging functionality into there.